pom.xml

```diff
@@ -0,0 +1,579 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Ingest API</name>
+    <inceptionYear>2019</inceptionYear>
+    <organization>
+        <name>Connexta</name>
+    </organization>
+    <licenses>
+        <license>
+            <name>GNU Lesser General Public v3</name>
+            <url>http://www.gnu.org/licenses/lgpl.html</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <groupId>com.connexta.ingest</groupId>
+    <artifactId>ingest-api</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <properties>
+        <root.package>com.connexta.ingest</root.package>
+
+        <!--  Default URL properties -->
+        <codice.scm.connection.url />
+        <snapshots.repository.url />
+        <reports.repository.url />
+
+        <!-- Build properties -->
+        <java.version>8</java.version>
```

Comment:
✏️ Should switch to 11

Reply:
Actually, I suggest we remove the Java 8 profile and move all Java11 profile properties back to the pom The only reason I had the 2 profiles is because the discovery api had to support both at the moment.

Reply:
The next commit won't have anything Java 8 related